docs/tutorials/get-started/quickstarts/control-motor.md

```diff
@@ -0,0 +1,169 @@
+
+{{< /expand >}}
+
+{{< expand "Step 3: Configure a motor" >}}
+
+[Add a motor component](/components/motor/), such as a [gpio motor](/components/motor/gpio/).
+Ensure your motor, motor driver, and board are properly connected.
+
+![The CONFIGURE tab of the Viam app populated with a configured gpio motor.](/components/motor/gpio-config-ui.png)
+
+{{< /expand >}}
+
+{{< expand "Step 4: Choose how you will control the motor" >}}
+
+You can control your motor directly from the Viam app, using the mobile app, or programatically.
+
+1. Control from the app
+
+   Navigate to your machine's **CONTROL** tab in the Viam app and use the **Power %** slider to set the motors speed.
+   Use the **Backwards** and **Forwards** buttons to change the direction.
+
+   {{<gif webm_src="/get-started/quickstarts/motor-control.webm" mp4_src="/get-started/quickstarts/motor-control.mp4" alt="Using the slider, Backwards, and Forwards buttons on the Viam app to control the direction and speed of a configured motor" class="alignright"  min-height="750px">}}
+
+2. Control from the mobile app
+
+   You can use [the Viam mobile app](/fleet/#the-viam-mobile-app) to control your motor's speed and direction directly from your smart device.
+
+   Open the Viam mobile app and log in to your account.
+   Select the location where your machine is assigned.
+   Choose your machine from the list and use the mobile interface to adjust the motor settings.
+
+   Select the location that your machine is assigned to from the **Locations** tab.
+
+   {{<gif webm_src="/get-started/quickstarts/mobile-app-motor-control.webm" mp4_src="/get-started/quickstarts/mobile-app-motor-control.mp4" alt="Using an example machine on the Viam mobile app to set the direction and speed of a configured motor using the slider on the user interface" max-height="50px" max-width="200px" clear="both" class="alignright">}}
+
+    <div style="clear: both;"></div>
+
+3. Control programatically
+
+   You can use the following code to control the motor's speed and direction using your preferred SDK:
+
+{{< tabs >}}
+{{% tab name="Python" %}}
+
+```python
+import asyncio
+
+from viam.robot.client import RobotClient
+from viam.rpc.dial import Credentials, DialOptions
+from viam.components.motor import Motor
+
+async def connect():
+    opts = RobotClient.Options.with_api_key(
+        # Replace "<API-KEY>" (including brackets) with your machine's api key
+        api_key='<API-KEY>',
+        # Replace "<API-KEY-ID>" (including brackets) with your machine's api key id
+        api_key_id='<API-KEY-ID>'
+    )
+    return await RobotClient.at_address('my-machine-main.1ye34y6p21.viam.cloud', opts)
+
+async def main():
+    machine = await connect()
+
+    print('Resources:')
+    print(machine.resource_names)
+
+    # motor-1
+    motor_1 = Motor.from_robot(machine, "motor-1")
+    motor_1_return_value = await motor_1.is_moving()
```

Comment:
can you use a different method here? This will not move the motor but just return that it's not moving.

Reply:
Same for go